examples/advanced/pi-lluminate.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python
+
+# Use the Enviro pHAT to check how much light is in a room, then trigger a webhook to control
+# Philips Hue lights via IFTTT. By Wesley Archer (@raspberrycoulis).
+
+# Import the relevant modules
+
+import requests
+import time
+import datetime
+import os
+from envirophat import light
+
+# Get the current time for displaying in the terminal.
+def whats_the_time():
+    now = datetime.datetime.now()
+    return (now.strftime("%H:%M:%S"))
+
+# The function to turn off the lights. Sends a webhook to IFTTT which
+# triggers Hue. Replace the trigger word and tokens from your account.
+
+def turn_off():
+    requests.post("https://maker.ifttt.com/trigger/{TRIGGER_WORD}/with/key/{TOKEN_GOES_HERE}")
```

Comment:
You should interpolate these variables, rather than manually insert them into each URL. IE:

```python

TRIGGER_WORD = "YOUR CHOSEN TRIGGER WORD GOES HERE"
TOKEN = "YOUR IFTTT TOKEN GOES HERE"

requests.post("https://maker.ifttt.com/trigger/{trigger_word}/with/key/{token}".format(trigger_word=TRIGGER_WORD, token=TOKEN))
```

Then if you want to get really fancy you can grab these values from `sys.argv` or `os.environ` and be able to run your code with arguments:

```
./pi-lluminate.py TOKEN TRIGGER_WORD
```

or with environment variables:

```
IFTTT_TOKEN="TOKEN" IFTTT_TRIGGER_WORD="TRIGGER WORD" ./pi-lluminate.py
```

Reply:
Yep, makes sense. I prefer the option for including the trigger word and token within the script, as it would be more "stand-alone".